### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by Irrreny**
Ivan, i remember you said you want to make footer page with us today, but i wanted to try to make it by myself first.
So here is the screenshot (remember, its just a try :) )
image

### CODE REVIEW

**Comment by IvanCeban** [comment]
WoW!

**Review by IvanCeban — COMMENTED** [review]
Wow! I'm impressed! Great job!
Let me do the code review after today's Lesson, but I bet there will be not much to comment here.

**Comment by Irrreny** [comment]
I did it completely different though, so looking forward to your comments :)

File: index.html [hunk 1]
```diff
@@ -69,33 +81,43 @@
...
+            padding: 20px;
+           }
+           .smaller-text {
+            text-align: center;
+            font-size: 20px;
+            font-weight: lighter;
+            margin: 0;
+            padding: 20px;
            }
            .call-to-action {
```
Comment (IvanCeban) [inline index.html:107]: good naming for button. I see you did some research about it

File: index.html [hunk 2]
```diff
@@ -164,14 +363,121 @@
...
+                <div class="card-header">
+                <img class="card-icon" src="img/icons/award.svg" alt="Award icon">
+                    <h5 class="card-header-text">Web Development</h5>
+                    </div>
+                    <p class="card-content">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+            </div>
+        </div>
+        </section>
+
+        <section class="footer">
```
Comment (IvanCeban) [inline index.html:427]: use a semantic `<footer>` tag for footer

File: index.html [hunk 3]
```diff
@@ -164,14 +363,121 @@
...
+            <div class="footer-header">
```
Comment (IvanCeban) [inline index.html:428]: Please pay attention that according the design footer content is aligned with the rest of the sections content. 
So I highly recommend you to wrap all footer content into a `container` same as the content of all the pages. 
You've set up padding 195px everywhere, but this is not flexible solution.

File: index.html [hunk 4]
```diff
@@ -142,7 +164,184 @@
...
+        }
+        .why-choose-us-text {
+            font-weight: 500;
+            line-height: 20px;
+            width: 470px;
+            text-align: center;
+        }
+        .grid-cards {
+            display: grid;
+            grid-template-columns: repeat(3,328px);
```
Comment (IvanCeban) [inline index.html:206]: Do not set fixed size to your cards. Let them take as much space as they can. Limit only max width of the container

File: index.html [hunk 5]
```diff
@@ -164,14 +363,121 @@
...
+                    <a href="#" class="footer-link">Customers</a>
+                    <a href="#" class="footer-link">API</a>
+                </div>
+                <div class="footer-info">
+                    <h5 class="footer-header">Get in touch</h5>
+                    <div class="footer-subscribe">
+                    <input type="email" class="footer-email" name="email" placeholder="Your Email" pattern="[a-zA-Z0-9._%+-]+@[a-z0-9.-]+\.[a-zA-Z]{2,4}" required>
+                    <input class="footer-button" type="submit" value="Subscribe">
+                    </div>
+                    <p style="font-size: 12px; color: #BDBDBD; margin: 0;">Please insert your email</p>
```
Comment (IvanCeban) [inline index.html:475]: inline style :(

File: index.html [hunk 6]
```diff
@@ -164,14 +363,121 @@
...
+                    <h5 class="footer-header">Resources</h5>
+                    <a href="#" class="footer-link">IOS &#38; Android </a>
+                    <a href="#" class="footer-link">Watch a Demo</a>
+                    <a href="#" class="footer-link">Customers</a>
+                    <a href="#" class="footer-link">API</a>
+                </div>
+                <div class="footer-info">
+                    <h5 class="footer-header">Get in touch</h5>
+                    <div class="footer-subscribe">
+                    <input type="email" class="footer-email" name="email" placeholder="Your Email" pattern="[a-zA-Z0-9._%+-]+@[a-z0-9.-]+\.[a-zA-Z]{2,4}" required>
```
Comment (IvanCeban) [inline index.html:472]: Would be better to wrap your input and button into a form

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Great code structure and good namings!
Please pay more attention to content alignment (I mean use container for any section content)

Also avoid fixed sizes. Always keep in mind that your website should be responsive

**Pull request closed without merging**